README.md

```diff
@@ -433,6 +433,7 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 * [letsencrypt-win-simple](https://github.com/Lone-Coder/letsencrypt-win-simple) - A Simple ACME Client for Windows.
 * [OctoLinker](https://github.com/OctoLinker/browser-extension) - Navigate through `projects.json` files efficiently with the OctoLinker browser extension for GitHub.
 * [Opserver](https://github.com/opserver/Opserver) - Stack Exchange's Monitoring System.
+* [posh-dotnet](https://github.com/bergmeister/posh-dotnet) - `PowerShell` tab completion for the [dotnet CLI](https://github.com/dotnet/cli).
```

Comment:
@bergmeister please resolve conflict then I will merge your pr. Thank you

Reply:
@ngohungphuc Thanks. The merge conflict is resolved now. It was a trival whitespace or carriage return conflict that TortoiseGit could auto-resolve for me.